mesheryctl/internal/cli/root/system/start.go

```diff
@@ -301,6 +302,43 @@ func start() error {
 		if err != nil {
 			return err
 		}
+
+		operatorURL := "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/manifests/default.yaml"
+
+		operatorManifest, err := meshkitutils.ReadFileSource(operatorURL)
+		if err != nil {
+			return err
+		}
+
+		err = utils.ApplyManifest([]byte(operatorManifest), client, true, false)
+		if err != nil {
+			return err
+		}
+
+		brokerURL := "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/samples/meshery_v1alpha1_broker.yaml"
+
+		brokerManifest, err := meshkitutils.ReadFileSource(brokerURL)
+		if err != nil {
+			return err
+		}
+
+		err = utils.ApplyManifest([]byte(brokerManifest), client, true, false)
+		if err != nil {
+			return err
+		}
+
+		meshsyncURL := "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/samples/meshery_v1alpha1_meshsync.yaml"
+
+		meshsyncManifest, err := meshkitutils.ReadFileSource(meshsyncURL)
+		if err != nil {
+			return err
+		}
+
+		err = utils.ApplyManifest([]byte(meshsyncManifest), client, true, false)
+		if err != nil {
+			return err
+		}
+
```

Comment:
We can move the `fetch- download- apply` steps of the manifest files to the already existing functions. 

That is, the level of abstraction in `start.go` should remain the same as before and all the changes made here would be additions to the existing functions in `platform.go`.

I am suggesting this because the only change here is that we have more manifest files and re-writing some stuff could be avoided and the abstraction in the `start.go` would be helpful.

Reply:
@navendu-pottekkat made these changes in my latest commit. Please review them.